src/main/java/net/imagej/ops/labeling/LabelingNamespace.java

```diff
@@ -88,6 +93,41 @@
 		return result;
 	}
 
+	// -- merge --
+
+	@OpMethod(op = net.imagej.ops.labeling.MergeLabeling.class)
+	public <L, I extends IntegerType<I>> ImgLabeling<L, I> merge(
+		final ImgLabeling<L, I> in1, final ImgLabeling<L, I> in2)
+	{
+		@SuppressWarnings("unchecked")
+		final ImgLabeling<L, I> result = (ImgLabeling<L, I>) ops().run(
+			net.imagej.ops.labeling.MergeLabeling.class, in1, in2);
```

Comment:
This should be `ops().run(Ops.Labeling.Merge.class, in1, in2);`. That way any op which implements the `Ops.Labeling.Merge` interface could be matched, as opposed to always using `net.imagej.ops.labeling.MergeLabeling`. 

This should also be changed in your other `merge(...)` namespace methods.